test/sim/items/ejectpack.js

```diff
@@ -128,7 +128,7 @@ describe(`Eject Pack`, function () {
 		assert.species(battle.p2.active[1], 'Wynaut');
 	});
 
-	it.skip(`should not trigger until after all entrance abilities have resolved during simultaneous switches`, function () {
+	it(`should not trigger until after all entrance abilities have resolved during simultaneous switches`, function () {
```

Comment:
The "important" behavior is working here (Drought triggered before the Pokemon switches), but Eject Pack itself is still triggering. That behavior is beyond the scope of this PR, so I would just change the test name to something like "should not prevent entrance Abilities from resolving during simultaneous switches".